### PR TITLE
Gradle: findbugs and sonarqube support

### PR DESCRIPTION
**Pull request by wischweh**
I have configured gradle to enable static code analysis with findbugs and sonarube. You still have to edit gradle.properties for the sonar server which you want to use. I can provide you with my personal configuration for evaluation

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -17,3 +17,13 @@
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 android.useDeprecatedNdk=true
+
+# General Sonar Configuration for project
+systemProp.sonar.sources=./src/main/java
+systemProp.sonar.scm.provider=git
+systemProp.sonar.projectName="Qabel :: Qbelbox :: App"
```
Comment (jan-schreib) [inline gradle.properties]: Typo?

**Comment by audax** [comment]
Thank you very much, I'll integrate this soon.

I guess looking at the output of android Lint wouldn't hurt either.

**Comment by audax** [comment]
Can one of the admins verify this patch?

**Comment by Rehwanne** [comment]
add to whitelist

**Pull request closed without merging**